task/backend/coordinator/task_coordinator.go

```diff
@@ -59,8 +59,16 @@ func (t SchedulableTask) Offset() time.Duration {
 
 // LastScheduled parses the task's LatestCompleted value as a Time object
 func (t SchedulableTask) LastScheduled() time.Time {
-	tm, _ := t.LatestCompletedTime()
-	return tm
+	if !t.LatestScheduled.IsZero() {
```

Comment:
need to check for nil since t.LatestSheduled is a pointer, alternatively this would be fine if we just made t,LatestScheduled not a pointer.

Reply:
The LatestScheduled is not a pointer. But on a taskUpdate it is a pointer so we have the option to not update latestScheduled.